lib/adapter.js

```diff
@@ -7732,6 +7766,17 @@ function Adapter(options) {
         }
     });
 
+    const pluginSettings = {
+        scope: 'adapter',
+        namespace: `system.adapter.${this.namespace}`,
+        logNamespace: this.namespaceLog,
+        log: logger,
+        iobrokerConfig: config,
+        parentPackage: this.pack
+    };
+    this.pluginHandler = new PluginHandler(pluginSettings);
+    this.pluginHandler.addPlugins(this.ioPack.common.plugins, [this.adapterDir, __dirname]); // first resolve from adapter directory, else from js-controller
```

Comment:
is `__dirname` supposed to be the js-controller root directory? Because it is not here.

Reply:
In fact it is the resolve path to resolve the module. According to the docs the lib dir will also work ... they check for a node_modules here, not found ... go one up (which is controller base) ... still not found ... go one up  and so on. So should work good enough. Or do I miss something?

Reply:
PS: And yes it is the controller base dir ... __dirname should point to the dir of adapter.js ... or ?!

Reply:
Ok so you rely on the require behavior, got it.